src/libraries/System.Text.Json/System.Text.Json.SourceGeneration.Tests/TestClasses.cs

```diff
@@ -1,13 +1,22 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using System.Collections.Generic;
 using System.Text.Json.Serialization;
 
+using TestsNamespace = System.Text.Json.SourceGeneration.Tests;
+
+[module: JsonSerializable(typeof(TestsNamespace.RepeatedTypes.Location))]
```

Comment:
Any reason why not "assembly:" instead?

Reply:
Per offline discussion there's really no reason to since the natural application in C# is `assembly`, I'll make a PR changing this.